src/terraform/examples/vfxt/opencue/pre-existing-azure-blob/moanaislandscenescripts/renderAll.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# /datadrive/pbrt/release/pbrt -outfile /home/AzureUser/island.png island.pbrt
```

Comment:
should this be commented out?  I see you have it as an included file

Reply:
This is just a sample script showing the command to start the renders, it's not a script you should execute unless running on a single VM.